blocks.js

```diff
@@ -11369,8 +11369,10 @@ MultiArgMorph.prototype.init = function (
 
     this.add(arrows);
 
-    // create the minimum number of inputs
-    for (i = 0; i < this.minInputs; i += 1) {
+    // create atleast one input
```

Comment:
Why is the default behavior changing for the MultiArgMorph? Shouldn't this just be on the morph used for the MultiHintArgMorph? Are we sure that changing the behavior here won't have any unexpected side effects in other uses of the MultiArgMorph?